fact/src/config.rs

```diff
@@ -20,4 +20,8 @@ pub struct FactConfig {
     /// Whether a small healthcheck probe should be run
     #[arg(long)]
     pub healthcheck: bool,
+
+    /// Whether to perform a pre flight check
+    #[arg(long, default_value_t = false)]
```

Comment:
[nit] For booleans the default is false, so we can skip this last part
```suggestion
    #[arg(long)]
```